packages/core/src/lib/TableNew/TableNew.stories.tsx

```diff
@@ -0,0 +1,224 @@
+};
+
+export const CustomLoader = Template.bind({});
+CustomLoader.args = {
+    header,
+    data: data,
+    pageSize,
+    maxPages,
+    isLoading: true,
+    customLoadingContent: (
+        <div>
+            <Loading size={30} text="Fetching..." spinnerColor={color.navy40} />
+        </div>
+    ),
+};
+
+export const HighlightTableRow = Template.bind({});
+HighlightTableRow.args = {
+    header,
+    data: data,
+    pageSize,
+    maxPages,
+    hover: true,
+    hoverBackgroundColor: color.navy20,
+};
+
+export const CustomLoadingSkeleton = Template.bind({});
+const EmptyRowsForSkeletonTable = () => (
+    <div style={{ width: '100%', height: '100%' }}>
+        {[...Array(6)].map((el, i) => (
+            <Skeleton theme="subtitle" width="30%" />
+        ))}
+    </div>
+);
+CustomLoadingSkeleton.args = {
+    header,
+    data: data,
+    pageSize,
+    maxPages,
+    isLoading: true,
+    customLoadingContent: (
+        <div
+            style={{
+                display: 'flex',
+                flexDirection: 'column',
+                width: '100%',
+                height: '100%',
+            }}
+        >
+            <EmptyRowsForSkeletonTable />
+            <EmptyRowsForSkeletonTable />
+            <EmptyRowsForSkeletonTable />
+        </div>
+    ),
+};
+
+export const SyncTableExample = Template.bind({});
+
+SyncTableExample.args = {
+    header: syncHeader,
+    data: syncData,
+    pageSize,
+    maxPages,
+    alignCellItems: 'center',
+    isColumnSortable: [false, true, true, false],
+};
+
+export const CustomTable: ComponentStory<typeof TableNew> = (args) => (
+    <TableNew {...args} />
+);
+
+CustomTable.args = {
+    header,
+    pageSize: 5,
+    maxPages,
+    isColumnSortable: [false, true, false, false],
+    data: customTableMockData,
+    tableBackgroundColor: 'lightblue',
+    headerTextColor: 'orange',
+    headerBgColor: 'blue',
+    isScrollableOnOverflow: false,
+    customDataComponent: (
+        <div
+            style={{
+                background: 'white',
+                width: '100%',
+                height: '200px',
+                textAlign: 'center',
+                display: 'flex',
+            }}
+        >
+            <h1 style={{ margin: 'auto' }}>Some Cool Image</h1>
+        </div>
+    ),
+};
+
+export const SingleColumnTable = Template.bind({});
+
+const TextStyled = styled.div`
+    width: 100%;
+    padding: 10px;
+    background-color: ${color.aero20};
+`;
+SingleColumnTable.args = {
+    header: ['Address'],
+    data: [
+        [<TextStyled>0xe922879D89E3a8D2f2D51EC8590AEF13216f9E7a</TextStyled>],
+        [<TextStyled>0xe922879D89E3a8D2f2D51EC8590AEF13216f9E7a</TextStyled>],
+        [<TextStyled>0xe922879D89E3a8D2f2D51EC8590AEF13216f9E7a</TextStyled>],
+        [<TextStyled>0xe922879D89E3a8D2f2D51EC8590AEF13216f9E7a</TextStyled>],
+    ],
+    pageSize: 10,
+    noPagination: true,
+    cellPadding: '0px',
+    customTableBorder: '1px solid black',
+};
```

Comment:
I would suggest adding a new story that shows how this new version solves the problems with the old one 😄 

It would also you understand if any more tweaks are needed.